spec/integration/subscribers_auth_token_spec.rb

```diff
@@ -35,7 +35,7 @@
     it "sends an email with the correct token" do
       post path, params: params
       expect(Email.count).to be 1
-      token = Email.last.body.match(/token=([^&)]+)/)[1]
+      token = Email.last.body.match(/token=([^&\n]+)/)[1]
```

Comment:
I'm not really clear why this regex isn't matching [this line](https://github.com/alphagov/email-alert-api/pull/1175/files#diff-24f45f86c97784ff32a8a8f2ff9e604eR48). However it's aimed at stripping the token off of the URL. We've changed it from some markdown to something that fits onto a newline.

By dropping a debugger here and taking the `request.body` into rubular. I got the following to work.

Reply:
Hmmm weird - wouldn't expect the token to have a `\n` char in it

Reply:
it's not the token, it gets the entire body out of the Email.last call then extracts the token from the link we send the user.

I think the test then expects we should be able to encrypt/decrypt that token to prove it's valid